flowredux/src/commonMain/kotlin/com/freeletics/flowredux/dsl/internal/StartStatemachineOnEnterSideEffectBuilder.kt

```diff
@@ -52,10 +52,12 @@ internal class StartStatemachineOnEnterSideEffectBuilder<SubStateMachineState :
                                 // dispatch the incoming actions to the statemachine
                                 coroutineScope {
                                     launch {
+                                        actionMapper(action.action)?.let {
```

Comment:
I personally don't like the `?.let { ... }` syntax. for me a simple `if action != null` check is more readable. 

Just a personal opinion. not a blocker.